### PR TITLE
KAFKA-9343: Add ps command for Kafka and zookeeper process on z/OS.

### PR DESCRIPTION
**Pull request by zshuo**
KAFKA-9343: Make Kafka shell runnable on z/OS

Finished both unit and integration test, and check

### Committer Checklist (excluded from commit message)
- [x] Verify design and implementation 
- [x] Verify test coverage and CI build status
- [x] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by zshuo** [comment]
@mimaison Could you please help review this? Thanks!

**Comment by mimaison** [comment]
@zshuo Thanks for the PR. This change are a bit "heavy-handed" considering z/OS is a niche platform for Kafka.

Instead of updating the Kafka scripts, it's probably easier to update the bash version in your environment. You can install bash 4.2 and the scripts should work fine once converted to EBCDIC.

**Comment by zshuo** [comment]
@mimaison Could you please help review again? Thanks!

File: bin/kafka-server-stop.sh [hunk 1]
```diff
@@ -14,7 +14,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 SIGNAL=${SIGNAL:-TERM}
-PIDS=$(ps ax | grep -i 'kafka\.Kafka' | grep java | grep -v grep | awk '{print $1}')
+
+if [ -z $JOBNAME ];then
```
Comment (mimaison) [inline bin/kafka-server-stop.sh]: Can we move this `if` block inside the `if [[ $(uname -s) == "OS/390" ]] ; then` block?

File: bin/zookeeper-server-stop.sh [hunk 2]
```diff
@@ -14,7 +14,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 SIGNAL=${SIGNAL:-TERM}
-PIDS=$(ps ax | grep java | grep -i QuorumPeerMain | grep -v grep | awk '{print $1}')
+
+if [ -z $JOBNAME ];then
+        JOBNAME="ZKEESTRT"
+fi
```
Comment (mimaison) [inline bin/zookeeper-server-stop.sh]: Can we move this `if` block inside the `if [[ $(uname -s) == "OS/390" ]] ; then` block?

**Review by mimaison — CHANGES_REQUESTED** [review]
Thanks for the update. I've left a couple of comments

**Comment by zshuo** [comment]
Thanks, it is a good suggestion, @mimaison, I updated.

**Review by mimaison — APPROVED** [review]

**Pull request merged**